Fixes #384

ko-lja opened the pull request.

Ghostipedia committed:
Buffed the hell out of Arboreal Growth Facility

Ghostipedia committed:
Buffed Nether sediment to yield more sediment per craft

Ghostipedia committed:
ALL of IV Bloodmagic progression for testing

 (Won't Work without Custom CosmicCore V0.6.3)

theMegaTech committed:
Apoth manasteel affix fix (#346)

* Fix manasteel and electric wirecutter affixes

* fuck i forgor elementium and terrasteel are also there

Ghostipedia committed:
Fix Rocket Plates not having ITNT

Ghostipedia committed:
The Return of ME Input hatches/busses

Ghostipedia committed:
LCG and ECG are now craftable

Ghostipedia committed:
divided the potent mana cost of terrasteel boosting in the EBF by 10

Ghostipedia committed:
Apothosis HellFoundry crafts

Ghostipedia committed:
Merge branch '0.6.3-dev' of https://github.com/Frontiers-PackForge/CosmicFrontiers into 0.6.3-dev

Ghostipedia committed:
QB Fixes

Ghostipedia committed:
Blood Orb Frames

Ghostipedia committed:
adding circuit tag to stuff that didn't have yeah uh huh huh mbmmmmmbmgbfmbmmfmfmfmbmdfdm,dfm,dfmfmfm BAAAA

Ghostipedia committed:
Gregtech

Ghostipedia committed:
T2 Blorb

Ghostipedia committed:
Assembly Line Trophy

Ghostipedia committed:
Ilmenite & Copper Mars Veins (Need more veins)

Ghostipedia committed:
The Stuff for T2 Blood Orb+Slate

Ghostipedia committed:
Use CosmicCore's triphenylphosphine

Ghostipedia committed:
Naquahine Materialset

Ghostipedia committed:
Add Ore Properties to Rutile/Titanium/U238

Ghostipedia committed:
Im Groving It

Ghostipedia committed:
Frontiers ME Pattern Buffer Recipe

Ghostipedia committed:
Made High-Grade Solder use Circ 17 vs 31???????

AWildVolt committed:
removes a lot of sophisticated stuff from jei+ adds recipes for smithing/refill upgrade and deco table (#357)

* stuff

* Update SophisticatedMods.js

* forgor

JuiceyBeans committed:
Stonecutting for marble and red granite (#356)

JuiceyBeans committed:
Replace vanilla bricks with coke/firebricks in steam age recipes (#355)

* Replace vanilla bricks with coke oven bricks

* Replace other steam recipes' bricks with equivalent

To match textures

AWildVolt committed:
gold > rose gold for most (non apparatus) recipes (#354)

* gold > rose gold for most (non apparatus) recipes

* forgot one

AWildVolt committed:
Fixes #351 (#353)

theMegaTech committed:
Improve the chance of getting the gold fern (#347)

AWildVolt committed:
remove tiny tnt+other redundant ae recipes (#345)

LaplaceRungeLenz committed:
LittleTiles recipes gregification (#344)

Basically using iron ingot/plate and LV components.If there''s any error or further request,feel free to ping me in discord,then i'll try to fix it.

Ghostipedia committed:
Gregtech

Ghostipedia committed:
Merge branch '0.6.3-dev' of https://github.com/Frontiers-PackForge/CosmicFrontiers into 0.6.3-dev

Ghostipedia committed:
No More Salt Dupe

 Im gonna time San out for a week again

Ghostipedia committed:
IV / LUV Questwork 1

Ghostipedia committed:
Tweaked Loading Msg for OreRemoval File

Ghostipedia committed:
IM STUPID AS SHIT JESUS CHRIST ON A BIKE (#364)

* FUCK I TARGETED MASTER

* AUUUUGH FUCK

Ghostipedia committed:
I hate this platform STAY RENAMED

Ghostipedia committed:
Bloodmagic Chalks, recipe removals, some other random shit

Ghostipedia committed:
Renamed Synth Netherite to Pthanterum

Ghostipedia committed:
Rhenium Extraction in the wrong file 📢

Ghostipedia committed:
Fix OnIoN

Ghostipedia committed:
Make Computer and HighPower Casings use Rhenium

Ghostipedia committed:
Goodbye Plasma Turbine

Ghostipedia committed:
Buff Star Gasoline to be 4000eu/mb

Ghostipedia committed:
All <5 Grading Catalysts Are LuV

Ghostipedia committed:
More Bloodmagic work

Ghostipedia committed:
trinium/nq+/nq* decomp due to chain differences

Ghostipedia committed:
Ritual Diviner

Ghostipedia committed:
misc material stuff

Ghostipedia committed:
Guhg changes

Ghostipedia committed:
Feed the Frontiert

Ghostipedia committed:
T5 Slate Prep

Ghostipedia committed:
Finally Rename them to Grading Catalyst

Ghostipedia committed:
T5 and T6

Ghostipedia committed:
Tau Berry Recipes

Ghostipedia committed:
Where tf did bees go

Ghostipedia committed:
Replaced KubeJS Neutronite with CosmicCore Neutronite

Ghostipedia committed:
BM Tweaks

Ghostipedia committed:
First Synthetic Pthanterum

Ghostipedia committed:
Naq and Rhenium Tweaks

JuiceyBeans committed:
Food group fixes (#350)

* I cannot be bothered to learn how to rebase a closed PR. Woe, squashed commit upon ye

* Blueberries don't work with seasons for some reason. Lmao

* Fix veg soup recipe

* Raw meat tag unification

* Add ground pork to raw pork tag

* Use FD skillet instead of Croptopia pan

* Salt unification

* Requested changes

JuiceyBeans committed:
Add icon for log quest to prevent smart filter lag (#358)

JuiceyBeans committed:
Update sapling shop quest to include mangrove and azalea (#361)

AWildVolt committed:
Apoth tomes & misc shelves recipes + seashelf "fix" (#363)

* fix seashelf recipe+tomes+misc shelf recipes

* potions are dumb

* rectifier recipes+upgraded hell/seashelf recipes

* last of the apoth recipes

* im dumb lmao...

* i... im tired is my excuse

AWildVolt committed:
make the xp reward quest not a liar (#372)

AWildVolt committed:
bookshelf fixes (#367)

* remove non assembler recipes for all bookshelf

* readded bookshelf conversion

removing wasn't needed :gpwide:

theMegaTech committed:
gold sun fern + affrit (#377)

LaplaceRungeLenz committed:
Create Rehooked.js (#376)

AWildVolt committed:
drill parity idk (#375)

AWildVolt committed:
lmao (#373)

Ghostipedia committed:
Im Questing it

Echoloquate committed:
Trinium mini line (#383)

* FUCK I TARGETED MASTER

* AUUUUGH FUCK

* Add recipe for magnetic iron bolt (for nature's compass) (#378)

* Add recipe for magnetic iron bolt (for nature's compass)

Magnetic components PR made it so crafting with an item removes magnetism. Add hand crafting recipe for magnetic iron bolt

* Oops

* Added trinium metallic powder processing line

* Removed comments

---------

Co-authored-by: Ghostipedia <Ghostipedia@gmail.com>
Co-authored-by: Karthi <75553966+JuiceyBeans@users.noreply.github.com>
Co-authored-by: Ghostipedia <46772882+Ghostipedia@users.noreply.github.com>

Ghostipedia committed:
misc tweaks and removals

Ghostipedia committed:
Chemvat Casing Recipes

Ghostipedia committed:
Fix Missing Neutronite Recipes, LuV Combustion Gen

Ghostipedia committed:
LuV Soul Output Hatch

Ghostipedia committed:
Fusion Trophy :)

Ghostipedia committed:
Tree Multi Boosting for Nutrient Sol.

Ghostipedia committed:
Removed Hydrogen Peroxide because GTM made a version of it (that looks like water)

Ghostipedia committed:
Misc Naq Reactor Changes

Ghostipedia committed:
I swear to god if the concrete bucket returns again I will kill myself

Ghostipedia committed:
ZPM Parts

Ghostipedia committed:
Lang

Ghostipedia committed:
Nutrient Solution

Ghostipedia committed:
Trinavine Rocket Parts (T3)

Ghostipedia committed:
Demon Will Production

Ghostipedia committed:
Fix to ZPM Graphene Recipe

Ghostipedia committed:
Bunch of Crystal Circuit Tweaks

Ghostipedia committed:
Nylon-6,6 and Trinanylon-6,6

Ghostipedia committed:
Vitrius

Ghostipedia committed:
Fixes to Trinium Oxide Compound Process for ZPM

Ghostipedia committed:
Fixed Drygmy Grove

Ghostipedia committed:
Rhenium, Trinavine, Neutronium and Starmetal Casings

Ghostipedia committed:
ZPM and LUV Assline Recipe Tweaks

Ghostipedia committed:
Trinavine Rocket Part Items

Ghostipedia committed:
GFC Structure Refactor

Ghostipedia committed:
Fixed a Typo in KaptonS

Ghostipedia committed:
Trinium Compound Materials

AWildVolt committed:
quick create fixes (#385)

AWildVolt committed:
adds a gt recipe for create deco industrial iron (#391)

ArchEzekiel committed:
GT recipes for experience gems (#389)

Added recipe which melted down Ars Nouveau experience gems into Cognitium at equivalent rates. Added packing recipes to switch between gem types. made the greater gem recipe slight faster to incentivize packing the small gems into large ones first

Ghostipedia committed:
bottom

Ghostipedia committed:
UV Casing/Hull

Ghostipedia committed:
I made the fucking sun

Ghostipedia committed:
Neutronium Start

YoungOnionMC committed:
Neoprene stuff (#392)

Ghostipedia committed:
Bump EU voltages of all of Neoprene

Ghostipedia committed:
give crystal neutronium gas a proper temp

Ghostipedia committed:
God Why is Material Registry Dumb

Ghostipedia committed:
Large Spooling Machine Lang

Ghostipedia committed:
Ruridit casings

Ghostipedia committed:
Migrate Spooling Recipemap to cosmiccore

Ghostipedia committed:
Tweak Recipe for Virtue Essence Replication

Ghostipedia committed:
Merge branch 'main-1.20.1-Forge' into 0.6.3-dev

Ghostipedia committed:
Revert "Merge branch 'main-1.20.1-Forge' into 0.6.3-dev"

This reverts commit e35a84ec594050db5ddb5d5c01553c82e1c3bb0d, reversing
changes made to 04de84c5b6193ceab5d04ddc444846db783eb795.

Ghostipedia committed:
Crystal Circuit Impl

Ghostipedia committed:
Venus Veins

Ghostipedia committed:
Remove Venus Ad-Astra Ores

Ghostipedia committed:
Annealed ManaSteel Fixes

Ghostipedia committed:
Custom Ores

Ghostipedia committed:
Fix Venus World Gen Dying

Ghostipedia committed:
Screwing Around with Tinkers Tools Made of neutronite???

Ghostipedia committed:
Tinkers Construct has so many bugged recipes!

Ghostipedia committed:
Lang and Assets

Ghostipedia committed:
Mercury Vein Todos

Ghostipedia committed:
YET ANOTHER GRADING CATALYST BUFF

Ghostipedia committed:
Neutronium.

Ghostipedia committed:
Crystal Circuit Tweaks, New Casings

Ghostipedia committed:
Gut Fusion Recipes for rework

Ghostipedia committed:
Make Nylon a bit less asshole-y

Ghostipedia committed:
Thank you, AE2.

Ghostipedia committed:
Nqh Reactor Stuff

Ghostipedia committed:
🌃

Ghostipedia committed:
We

Ghostipedia committed:
SGM like 50% done

Ghostipedia committed:
All Other changes Done before Taking a Break

AWildVolt committed:
gregify ender buckets/bags (#400)

AWildVolt committed:
adds assembler recipes for FS upgrades (#404)

* adds assembler recipes for FS upgrades

* make framed drawer recipes cheaper

AWildVolt committed:
gregify shrinking device (#401)

* gregify shrinking device

* forgor 💀

AWildVolt committed:
change energy cell recipe to use any lv battery (#403)

AWildVolt committed:
adds sophisticated recipes for missing items. fixes some recipes. adds some recipes to assemblers (#405)

* stuff

* advanced jukebox recipes

AWildVolt committed:
change shelves required materials (#407)

AWildVolt committed:
adds recipes for netherite halberd (#408)

Spicierspace153 committed:
fix ae2certus inworld (#422)

Co-authored-by: spaceboi153 <Spicierspace153@users.noreply.github.com>

AWildVolt committed:
add distilled water recipe for washed magebloom (#426)

AWildVolt committed:
make cleaning hatch use any luv circuit (#427)

AWildVolt committed:
remove faucet recipe (#428)

AWildVolt committed:
add netherite block and related recipes (#432)

rinOfTheStars committed:
add sun fern recipes (#439)

they're basically identical to the ice fern ones minus the inputted and outputted ferns

Ghostipedia committed:
Return of the decaffeinated Queen

Ghostipedia committed:
Merge branch '0.6.3-dev' of https://github.com/Frontiers-PackForge/CosmicFrontiers into 0.6.3-dev

Ghostipedia committed:
Resolve Issue #438

Ghostipedia committed:
Fix Ad-Astra Rocket Plate Recipe being cooked

Ghostipedia committed:
Prismatic Lens and Uranium238 For Uraninite

Ghostipedia committed:
Misc Fixes - Last for now

serenibyss committed:
Remove fops dropping beef (#443)

Ghostipedia committed:
Add new Splash Texts (#444)

AWildVolt committed:
oat cutting fix ig? (#440)

txnug committed:
Add shapeless energy acceptor recipe (#446)

Ghostipedia committed:
A bunch of nonsense

Ghostipedia committed:
b

Ghostipedia committed:
Oh my god so much QB

Ghostipedia committed:
I am unsure what I am doing at this point!

Ghostipedia committed:
Stop space from muffling sound

Ghostipedia committed:
We Should Mixin EU into AE2

Ghostipedia committed:
Deny Bosses from Apoth Spawners

Ghostipedia committed:
Denied a bunch of mods from CarryOn to not kill the game

Ghostipedia committed:
GTCEU:MODERN Config tweaks

- Enable Hard Rods
- Enable Hard Redstone
- Disable GT Loot (Caused issues)
- Disable Hazards

Ghostipedia committed:
Kill EUp2p Entirely

We (CosmicCore) unsubbed EUp2p from the event bus w/ a mixin, disabling this config from MAE2 disables them from even being registered.
Hopefully this is enough deterence for EUp2p to go away

Ghostipedia committed:
Force Appropriate Gamerules & Remove Graves

- Keep Inventory is now a forced-default gamerule
- Instant Respawn is an forced-default gamerule
- Remove graves to stop duping

This is closer to the death-like behaviors i wish the character to have, so this works well

Ghostipedia committed:
For TCon stuff for 0.8.0

Ghostipedia committed:
Stop duplicating mana rods

Ghostipedia committed:
Improve Potent Mana Rates for Source Diesel!

srdr2k3 committed:
fix and gregify menril (#462)

srdr2k3 committed:
Add missing UG fungi recipes (#459)

txnug committed:
Update en_us.json / Fix ritual diviner research (#460)

* Update en_us.json

change industiral to industrial

* Update Gregtech_Assline.js

* Update Gregtech_Assline.js

srdr2k3 committed:
Remove CEu Bauxite Line (#458)

* There shall only be one

* There will be no survivors

* bauxite electrolysis kil

srdr2k3 committed:
Fix & Rework Netherite Line (#451)

Ghostipedia committed:
Platline Fixes

Ghostipedia committed:
The Sun.

Ghostipedia committed:
Merge branch '0.6.3-dev' of https://github.com/Frontiers-PackForge/CosmicFrontiers into 0.6.3-dev

Ghostipedia committed:
The pack parses recipes again... Kinda

Ghostipedia committed:
Gwagwa

srdr2k3 committed:
Add ITNT recipe for Entropinnyum (#466)

1 ITNT instead of 4 TNT

srdr2k3 committed:
add circuit to h2o2 recipe (#468)

WamenNudles committed:
Fixing LuV softlocks (#473)

Added recipe for Wrapped Demonic Slates
Changed the temp requirement for Naquadah Alloy from 7200K to 5900K, in EBF and ABS

Ghostipedia committed:
Shelf ULVCovers until update; replaced with 1 shard ec.

Ghostipedia committed:
Fixing missing CCore Blocks in recipes i think

Ghostipedia committed:
Why'd the lang break what

ko-lja committed:
Fixes #384

Ghostipedia committed:
Sure do wish we had datafixers for KubeJS

ko-lja committed:
Fixed #475 (#476)

Co-authored-by: Kolja <104643407+Thatguyello@users.noreply.github.com>

Ghostipedia commented:
Merge Conflicts, and potentially as a result incorrect recipes

ko-lja committed:
Fixes #330 (#478)

Remove reference to ethanol loop in quest

Ghostipedia committed:
Mrrp

Ghostipedia committed:
Merge branch '0.6.3-dev' of https://github.com/Frontiers-PackForge/CosmicFrontiers into 0.6.3-dev

Ghostipedia committed:
Temp Overhauls 1

Ghostipedia committed:
Oversized Interface granted 4096 Storage Cap

Ghostipedia committed:
AE2 Reads in EU Config

Ghostipedia committed:
Fancy Menu Tweaks

Ghostipedia committed:
No More Backpack Cheese

Ghostipedia committed:
Gregtech Config

Ghostipedia committed:
Yet Again wish KubeJS datafixer was real

Ghostipedia committed:
Misc Nonsesnse

Ghostipedia committed:
Revert "Shelf ULVCovers until update; replaced with 1 shard ec."

This reverts commit dd0b8ac4e40ca1dbb0fc44335a540ece1538444a.

Ghostipedia committed:
Revert the ULVCover Safeguard

AveryAndCompany committed:
fixes #482 (#484)

jurrejelle committed:
remove circuit requirements from red mud for bauxite line (#485)

srdr2k3 committed:
add parallel hatch to dryer multi (#486)

srdr2k3 committed:
add 1 step ae cable coloring (#487)

Ghostipedia committed:
guh

srdr2k3 committed:
no more nitric acid gain (#491)

srdr2k3 committed:
kill easy recipe (#490)

srdr2k3 committed:
Change programmed circuit 0 to 11 in create.js (#488)

* change programmed circuit 0 to 1

* change to 11 instead

srdr2k3 committed:
add batch mode to KJS multis that didn't have it (#489)

srdr2k3 commented on overrides/kubejs/server_scripts/Recipes/Create.js:
```diff
@@ -706,7 +706,7 @@ ServerEvents.recipes(event => {
   })
   event.recipes.gtceu.assembler("create:mechanical_drill")
-    .itemInputs('2x create:andesite_casing', '2x gtceu:andesite_alloy_plate', 'gtceu:blue_steel_drill_head', 'gtceu:andesite_alloy_gear', 'ulvcovm:ulv_electric_motor')
+    .itemInputs('2x create:andesite_casing', '2x gtceu:andesite_alloy_plate', 'gtceu:steel_drill_head', 'gtceu:andesite_alloy_gear', 'ulvcovm:ulv_electric_motor')
```
Not a fan of this change. Unlike what the issue mentions, Blue Steel can be cooled in a chemical bath.
Create drills allow for really fast ways to mine out ore veins when spammed en masse.
The crafting table recipe should be changed to blue steel instead.

srdr2k3 committed:
death (#492)

Adds a recipe for the Superdense ME Energy Cell, and fixes/nerfs the Creative Energy Cell

Ghostipedia committed:
Tapestry Recipe

Ghostipedia committed:
HPCA Indicator

Ghostipedia committed:
Merge branch '0.6.3-dev' of https://github.com/Frontiers-PackForge/CosmicFrontiers into 0.6.3-dev

Ghostipedia committed:
Bwa Merge Conflict

Ghostipedia committed:
I hate GTM

srdr2k3 committed:
no more free anchors (#493)

ko-lja committed:
Merge branch '0.6.3-dev' into ko-lja-patch-1
